br: add faq - why oom when restore stats (#16781)
pingcap/docs: faq/backup-and-restore-faq.md

## After the recovery is complete, can I delete a specific table and then recover it again?

Yes, after deleting a specific table, you can recover it again. But note that, you can only recover tables that are deleted using the `DROP TABLE` or `TRUNCATE TABLE` statement, not the `DELETE FROM` statement. This is because `DELETE FROM` only updates the MVCC version to mark the data to be deleted, and the actual data deletion occurs after GC.

> ## Why does BR use a lot of memory when restore statistic data?

Reviewer: ```suggestion
## Why does BR use a lot of memory when restoring statistic data?
```

> Before v7.6.0, the statistic data and the table/schema information data are backed up into the same file, and are loaded together into memory when restore. Therefore, When the size of statistic is too large, BR would use a lot of memory.

Reviewer: ```suggestion
Before v7.6.0, the statistic data and the table/schema information data is backed up into the same file, and is loaded together into memory when restoring. Therefore, When the size of statistic is too large, BR will use a lot of memory.
```

> ## Why does BR use a lot of memory when restoring statistic data?
> 
> Before v7.6.0, the statistic data and the table/schema information data is backed up into the same file, and is loaded together into memory when restoring. Therefore, When the size of statistic is too large, BR will use a lot of memory.
> 
> Starting from v7.6.0, the statistic data is backed up into separate files. BR does not load statistic data of any table until BR starts to restore the table, which saves memory.

Reviewer: ```suggestion
### Why does BR take a lot of memory when restoring statistics information?

Before v7.6.0, the statistics data backed up by BR is stored together with the table information and loaded into memory during recovery. Therefore, when the backup statistics data is very large, BR needs to occupy a large amount of memory.

Starting from v7.6.0, the backup statistics is stored in a specific file separately. BR does not load statistic data of any table until BR starts to restore the table, which saves memory.
```

> Since v7.6.0, the statistic data is backed up into some individual files. BR wouldn't load statistic data of any tables until BR starts to restore the statistic data of the table, which saves memory.

Reviewer 2: ```suggestion
Since v7.6.0, the statistic data is backed up into some individual files. BR wouldn't load statistic data of any table until BR starts to restore the table, which saves memory.
```
